Userland/Libraries/LibWeb/CSS/CSSStyleDeclaration.cpp

```diff
@@ -235,9 +252,22 @@ bool PropertyOwningCSSStyleDeclaration::set_a_css_declaration(PropertyID propert
 String CSSStyleDeclaration::get_property_value(StringView property_name) const
 {
     auto property_id = property_id_from_string(property_name);
-    if (!property_id.has_value())
+
+    // check for a custom property and return early
+    if (!property_id.has_value() && is_a_custom_property_name_string(property_name)) {
```

Comment:
Custom property names never overlap with actual property names, so this can go before the `property_id` declaration.